Projects/App/Sources/AppDelegate.swift

```diff
@@ -49,7 +49,6 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
     }
 
     func logout() {
-        bag.dispose()
```

Comment:
Can we add here cancellables.removeAll() ?
It will mimic what that bad used to do